partcad/src/partcad/project.py

```
@@ -1889,6 +1717,14 @@
                 parameters += "</ul>\n"
                 columns += [parameters]
 
+            if not "images" in config and "desc" in config and "INSERT_IMAGE_HERE" in config["desc"]:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)